ibm/service/vpc/data_source_ibm_is_vpc_routing_table_routes.go

```diff
@@ -136,6 +136,54 @@ func DataSourceIBMISVPCRoutingTableRoutes() *schema.Resource {
 							Computed:    true,
 							Description: "Routing Table Route Nexthop Address or VPN Gateway Connection ID",
 						},
+						"next_hop_details": &schema.Schema{
```

Comment:
Please refer data_source_ibm_is_vpc_routing_table_route.go file the schema and use the same here

Reply:
implement the acceptance test case, attach the results..
Attach the terraform testing template and the result also here..